sale_multicompany_reporting_currency/models/sale_order.py

```diff
@@ -93,7 +98,6 @@ def _compute_amount_multicompany_reporting_currency(self):
             ):
                 to_amount = reporting_amount
             else:
-                to_amount = (
-                    reporting_amount / record.multicompany_reporting_currency_rate
-                )
+                inverse_rate = 1.0 / record.multicompany_reporting_currency_rate
+                to_amount = reporting_amount / inverse_rate
```

Comment:
Mathematically, to divide by the inverse rate is equivalent to multiplying by the original rate, so this snippet of code is equivalent to:
```suggestion
                to_amount = reporting_amount * record.multicompany_reporting_currency_rate
```
which (I believe) is easier to read